common/jdbc/build.gradle

```diff
@@ -36,13 +36,3 @@ bootJar {
 jar {
     enabled=true
 }
-
-apply plugin: 'scalaStyle'
-
-scalaStyle {
-    //configLocation = "$rootDir/gradle/conf/scalastyle_conf.xml"
-    configLocation = "${rootDir}/conf/scalastyle-config.xml"
-    includeTestSourceDirectory = true
-    source = "src/main/scala"
-    testSource = "src/test/scala"
-}
```

Comment:
Any reason to remove scala style plugin. I feel like it is better to have it, especially for future fixes.

Reply:
can't find one compatible with scala 2.12